src/sstruct_ls/fac_zero_stencilcoef.c

```diff
@@ -6,6 +6,7 @@
  ******************************************************************************/
 
 #include "_hypre_sstruct_ls.h"
+#include "boxloop.h"
```

Comment:
What's going on here?

Reply:
The files in `struct_mv` and `struct_ls` are more problematic. I've moved boxloop headers that contain the implementations out of `_hypre_struct_mv.h` into a new file called `boxloop.h`. So, whenever a file has `BoxLoop` call, `boxloop.h` has to be included.